### PR TITLE
Allow multiple releases to be deployed.

### PR DESCRIPTION
**Pull request by MG2R**
Also generalized the inclusion of dotdeb-wheezy.
Backwards compatibility has remained.

### CODE REVIEW

File: manifests/repo/debian.pp [hunk 1]
```diff
@@ -42,24 +43,30 @@
...
     location    => $location,
-    release     => $release,
     repos       => $repos,
     include_src => $include_src,
-    require     => Apt::Key['php::repo::debian'],
   }
 
   if ($dotdeb) {
-    # wheezy-php55 requires both repositories to work correctly
+    # Dotdeb requires the general 'wheezy' release as well.
```
Comment (fpletz) [inline manifests/repo/debian.pp]: Are you sure? Even on jessie? The instructions don't reflect this.

**Comment by fpletz** [comment]
Hmm, I don't quite understand what you mean with 'multiple releases'. `php::repo::debian` is still a class so there is just one apt source that's being deployed.

**Comment by MG2R** [comment]
Please ignore this pull request. I don't know what I was thinking when I wrote this, but I just rolled this back on my own environments.

**Comment by fpletz** [comment]
Thanks for the follow-up.

**Pull request closed without merging**